package.json

```diff
@@ -41,6 +41,7 @@
     "jsdoc-to-markdown": "^4.0.1",
     "mocha": "^5.0.5",
     "mochainon": "^2.0.0",
+    "mock-spawn": "0.2.6",
```

Comment:
Is there a specific reason for this version to be pinned? If not, a ^ range selector would be preferred here